appinventor/components/src/com/google/appinventor/components/runtime/util/NativeOpenStreetMapController.java

```diff
@@ -429,6 +465,10 @@ public void setMapTypeAbstract(MapType type) {
       case Terrain:
         view.setTileSource(TileSourceFactory.USGS_TOPO);
         break;
+      // case Terrain:
+      case Custom:
```

Comment:
Note that I don't think there is any way to set this right now from the designer since this PR doesn't add an option to YoungAndroidMapTypePropertyEditor. It is settable from the blocks though. I'm not sure that specific combination was the intention or not.

Reply:
It was not intentional and I couldn't figure out why it was not working from designer.  I will investigate `YoungAndroidMapTypePropertyEditor`, thank you.

Reply:
Thank you, that was exactly the hint that I needed.